src/v/kafka/server/quota_manager.h

```diff
@@ -36,6 +38,53 @@
 
 namespace kafka {
 
+using client_quota_clock = ss::lowres_clock;
```

Comment:
I'm somewhat averse to sticking using declarations at header scope like this. if you go with ben's suggestion to stick the probe instance behind a unique_ptr and init/deinit in mgr::start/stop, it probably gets a tiny bit simpler to wire up the clock type as well.

Reply:
Makes sense, done